src/NexusMods.App.UI/WorkspaceSystem/PanelTabHeader/PanelTabHeaderView.axaml

```diff
@@ -23,7 +23,7 @@
                 <StackPanel Orientation="Horizontal"
                             Classes="Spacing-1_5"
                             VerticalAlignment="Center">
-                    <icons:UnifiedIcon MaxSize="24" Size="24" x:Name="Icon"/>
+                    <icons:UnifiedIcon MaxSize="24" Size="18" x:Name="Icon"/>
                     <TextBlock x:Name="TitleTextBlock"
```

Comment:
Is this supposed to be `18`? It looks kinda small. Also, you forgot to change `MaxSize` to `18` as well.

![2024-04-29_12-59-12](https://github.com/Nexus-Mods/NexusMods.App/assets/16577545/2c192c08-3b31-4a9b-9d4f-738abe7235e2)

Reply:
18 is correct, why is MaxSize needed?

Reply:
> why is MaxSize needed?

Depending on the icon type, if `Size` differs from `MaxSize`, then the bounds will be set to `MaxSize` and the control will contain empty space. In this case, we know that the icon has a fixed size.